packages/lib/errors.ts

```diff
@@ -34,7 +35,7 @@ export function handleErrorsRaw(response: Response) {
     console.error({ response });
     return "{}";
   }
-  if (!response.ok && response.status < 200 && response.status >= 300) {
+  if (!response.ok || response.status < 200 || response.status >= 300) {
```

Comment:
status can't be simulatenously less than 200 and greater than 300. It should have been an OR.

Also, response.ok can independently decide error response.